twitter_scrooge/twitter_scrooge.bzl

```diff
@@ -140,7 +140,7 @@ def _gen_scrooge_srcjar_impl(ctx):
   # in order to generate code) have targets which will compile them.
   _assert_set_is_subset(only_transitive_thrift_srcs, transitive_owned_srcs)
 
-  path_content = "\n".join([_colon_paths(ps) for ps in [immediate_thrift_srcs, only_transitive_thrift_srcs, remote_jars, external_jars]])
+  path_content = "\n".join(["_" + _colon_paths(ps) for ps in [immediate_thrift_srcs, only_transitive_thrift_srcs, remote_jars, external_jars]])
```

Comment:
Can you extract this into a constant with a clear name? Something like PaddingToPreserverAruent or PaddingToSeparateArgumentToOwnLine? Names are different once I'm really not sure I got the point well enough...